decidim-participatory_processes/app/views/decidim/participatory_processes/participatory_processes/_promoted_process.html.erb

```diff
@@ -5,7 +5,7 @@
         <%= link_to participatory_process_path(promoted_process), class: "card__link" do %>
           <h2 class="card__title"><%= decidim_html_escape(translated_attribute(promoted_process.title)).html_safe %></h2>
         <% end %>
-        <%= decidim_sanitize_editor html_truncate(translated_attribute(promoted_process.short_description), length: 630, separator: "...") %>
+        <%= decidim_sanitize_editor_admin html_truncate(translated_attribute(promoted_process.short_description), length: 630, separator: "...") %>
```

Comment:
Same thing here. I think showing the embeds in the promoted processes is not necessary.

If you disagree, could you explain a concrete example of this?

Reply:
Same, reverted in update.